lib/SQL/Translator/Producer/PostgreSQL.pm

```diff
@@ -553,7 +570,10 @@ sub create_index
     }
     elsif ( $type eq NORMAL ) {
         $index_def =
-            'CREATE INDEX ' . $generator->quote($name) . ' on ' . $generator->quote($table_name) . ' ' . $field_names
+            'CREATE INDEX ' . $generator->quote($name) . ' on ' . $generator->quote($table_name) .
+            ($index_using ne '' ? ' ' . $index_using : '') .
```

Comment:
If you include the leading space when setting the value above, you can get rid of the ugly conditionals here.